From 7bbee6ac49f46319f8bda222ae67377252346704 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 6 Nov 2014 10:44:33 +0100
Subject: [PATCH] systemd: add 90-vconsole.rules

run systemd-vconsole-setup if fbcon device shows up

(cherry picked from commit 027565d0233f76edf434b63a847a717185d267e2)
---
 modules.d/98systemd/module-setup.sh | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index 015597f..f62f551 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -196,7 +196,12 @@ install() {
     inst_binary true
     ln_r $(type -P true) "/usr/bin/loginctl"
     ln_r $(type -P true) "/bin/loginctl"
-    inst_rules 70-uaccess.rules 71-seat.rules 73-seat-late.rules 99-systemd.rules
+    inst_rules \
+        70-uaccess.rules \
+        71-seat.rules \
+        73-seat-late.rules \
+        90-vconsole.rules \
+        99-systemd.rules
 
     for i in \
         emergency.target \
